src/AdafruitIO.h

```diff
@@ -12,6 +12,10 @@
 #ifndef ADAFRUITIO_H
 #define ADAFRUITIO_H
 
+#ifndef ADAFRUITIO_RUN_TIMEOUT
+#define ADAFRUITIO_RUN_TIMEOUT  60000
```

Comment:
Put in https://github.com/adafruit/Adafruit_IO_Arduino/blob/master/src/AdafruitIO_Definitions.h

Change to `ADAFRUITIO_MQTT_CONNECTION_TIMEOUT` to be more descriptive.

Reply:
AIO_MQTT_CONNECTION_TIMEOUT matches convention in definitions